yarn.lock

```diff
@@ -11445,7 +11344,32 @@ terriajs-cesium-widgets@5.0.0:
     nosleep.js "^0.12.0"
     terriajs-cesium "^8.0.0"
 
-terriajs-cesium@8.0.0, terriajs-cesium@^8.0.0:
+terriajs-cesium@8.0.1:
+  version "8.0.1"
+  resolved "https://registry.yarnpkg.com/terriajs-cesium/-/terriajs-cesium-8.0.1.tgz#2f7fb10d8f2d54258022a650d19fb1afe1a3a7d7"
+  integrity sha512-99sEp1DKC2g67QwRueE+EZxbOrI9nAw+LP5AsGBtJqr6cD+20Whkx3kMDweE4tjeBMWPjDMZbsbFbILbbSqqVw==
+  dependencies:
+    "@tweenjs/tween.js" "^23.1.1"
+    "@zip.js/zip.js" "^2.7.34"
+    autolinker "^4.0.0"
+    bitmap-sdf "^1.0.3"
+    dompurify "^3.0.2"
+    draco3d "^1.5.1"
+    earcut "^2.2.4"
+    grapheme-splitter "^1.0.4"
+    jsep "^1.3.8"
+    kdbush "^4.0.1"
+    ktx-parse "^0.6.0"
+    lerc "^2.0.0"
+    mersenne-twister "^1.1.0"
+    meshoptimizer "^0.20.0"
+    pako "^2.0.4"
+    protobufjs "^7.1.0"
+    rbush "^3.0.1"
+    topojson-client "^3.1.0"
+    urijs "^1.19.7"
+
+terriajs-cesium@^8.0.0:
```

Comment:
Shouldn't this "resolve" to 8.0.1, so there is a single version of terriajs-cesium?

Reply:
Good pickup, thats fixed now